src/cfi.rs

```diff
@@ -1912,11 +1912,8 @@ where
 
     fn push_row(&mut self) -> Result<()> {
         let new_row = self.row().clone();
-        if self.stack.push(new_row).is_some() {
-            Err(Error::CfiStackFull)
-        } else {
-            Ok(())
-        }
+        self.stack.try_push(new_row)
+                  .map_err(|_| Error::CfiStackFull)
```

Comment:
I haven't rustfmt'ed this. Maybe it won't like this line

Reply:
No worries.